benchmark_runner/workloads/workloads_operations.py

```diff
@@ -46,13 +46,15 @@ def __init__(self):
         self._es_port = self._environment_variables_dict.get('elasticsearch_port', '')
         self._es_user = self._environment_variables_dict.get('elasticsearch_user', '')
         self._es_password = self._environment_variables_dict.get('elasticsearch_password', '')
+        self._es_ssl = self._environment_variables_dict['elasticsearch_ssl'] == "true"
```

Comment:
I think it would be safer to have a stronger bool parsing,
currently, any non-empty string is `true`, including `no`, `false`, ...

I'll remove this `== "true"` as it's also arbitrary to exclude `True`

Reply:
OK

Reply:
I did an earlier PR to convert the environment variables to command line arguments; part of that was a common parser for true vs. false, and then store the boolean value rather than the string.  Perhaps after I get back from PTO I should revisit at least that part of it.